consts/event_type.py

```diff
@@ -3,6 +3,7 @@ class EventType(object):
     DISTRICT = 1
     DISTRICT_CMP = 2
     CMP_DIVISION = 3
+    CMP_SUBDIVISION = 5
```

Comment:
There (sorta) are. For last 2015/2016 CMP, there were 4 divisions, each with 2 "subdivision" to get the 8 "divisions". Some awards were done at the subdivision level, some at the division level. 

FMSAPI makes the distinction between division and subdivision, but we do all those manually anyway. So, I guess I should get rid of these for now, since the change to import CMP divisions will be better suited for a separate diff, anyways.